src/pages/instances/forms/CreateImageFromInstanceForm.tsx

```diff
@@ -0,0 +1,131 @@
+import { FC } from "react";
+import { LxdInstance } from "types/instance";
+import { useEventQueue } from "context/eventQueue";
+import { useFormik } from "formik";
+import { useToastNotification } from "context/toastNotificationProvider";
+import { createImageAlias, createImageFromInstance } from "api/images";
+import {
+  ActionButton,
+  Button,
+  Form,
+  Input,
+  Modal,
+} from "@canonical/react-components";
+import * as Yup from "yup";
+import { Link } from "react-router-dom";
+
+interface Props {
+  instance: LxdInstance;
+  close: () => void;
+}
+
+const CreateImageFromInstanceForm: FC<Props> = ({ instance, close }) => {
+  const eventQueue = useEventQueue();
+  const toastNotify = useToastNotification();
+
+  const notifySuccess = () => {
+    const published = (
+      <Link to={`/ui/project/${instance.project}/images`}>published</Link>
+    );
```

Comment:
```suggestion
    const created = (
      <Link to={`/ui/project/${instance.project}/images`}>created</Link>
    );
```
Maybe be a bit more neutral here, as the published flag might be false on the created image?